File: Exercicio revisão/Exercicio revisão/cliente.py
class cliente:
    def __init__(self, nome, email, senha):
        self.nome = nome
        self.email = email
        self.__senha = senha  

    def exibir_informacoes(self):
        print(f"Nome: {self.nome}\nEmail: {self.email}")

    def alterar_senha(self, nova_senha):
        self.__senha = nova_senha
        print("Senha alterada com sucesso.")

    def verificar_senha(self, senha):
        return self.__senha == senha

    @classmethod
    def criar_cliente(cls, nome, email, senha):
        return cls(nome, email, senha)

cliente1 = cliente.criar_cliente("Rafael", "rafael@gmail.com", "123")
cliente1.exibir_informacoes()
verificação = cliente1.verificar_senha("123")
print(verificação)
if verificação == False:
    print("Senha incorreta!")
else:
    cliente1.alterar_senha("456")  

File: Exercicio revisão/Exercicio revisão/cliente.java
public class cliente {
    private String nome;
    private String email;
    private String senha;  

    public cliente(String nome, String email, String senha) {
        this.nome = nome;
        this.email = email;
        this.senha = senha;
    }

    public void exibirInformacoes() {
        System.out.println("Nome: " + nome + "\nEmail: " + email);
    }

    public void alterarSenha(String novaSenha) {
        this.senha = novaSenha;
        System.out.println("Senha alterada com sucesso.");
    }

    public boolean verificarSenha(String senha) {
        return this.senha.equals(senha);
    }

    public static cliente criarCliente(String nome, String email, String senha) {
        return new cliente(nome, email, senha);
    }

    public static void main(String[] args) {
        cliente cliente1 = cliente.criarCliente("Rafael", "rafael@gmail.com", "123");
        cliente1.exibirInformacoes();
        System.out.println(cliente1.verificarSenha("senha123")); 
        cliente1.alterarSenha("456");  
    }
    
}
